pkg/grpccatalog/grpccatalog.go

```diff
@@ -108,6 +112,19 @@ func (r *GrpcCatalog) EnsureResources(key types.NamespacedName, displayName, pub
 	return nil
 }
 
+func setTimestampLabel(catsrc *olm.CatalogSource) {
```

Comment:
not 100% sure this is necessary, since we watch events on the GRPC connection and resync the catalogsource if the connection state changes. 

The grpc connection can't live through a pod rollout but this seems like a reasonable safeguard in case some events are missed / issues with the grpc conection watch.